src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -43,6 +43,8 @@
 def main(args):
 
     args.date = datetime.strptime(args.date, '%Y%m%d%H')
+    global qc
```

Comment:
very uncomfortable with setting `qc` to a global

and should only need to do it once so confused why it appears more than once?